service/history/replicationTaskExecutor_test.go

```diff
@@ -214,43 +210,6 @@ func (s *replicationTaskExecutorSuite) TestProcessTaskOnce_SyncActivityReplicati
 	s.NoError(err)
 }
 
-func (s *replicationTaskExecutorSuite) TestProcessTaskOnce_HistoryReplicationTask() {
-	namespaceID := uuid.New()
-	workflowID := uuid.New()
-	runID := uuid.New()
-	task := &replicationspb.ReplicationTask{
-		TaskType: enumsspb.REPLICATION_TASK_TYPE_HISTORY_TASK,
```

Comment:
you might want to rename REPLICATION_TASK_TYPE_HISTORY_V2_TASK -> REPLICATION_TASK_TYPE_HISTORY_TASK

Reply:
Probably will do it as a separate change.